language/heredoc_spec.rb

```diff
@@ -59,6 +59,22 @@
     s.encoding.should == Encoding::US_ASCII
   end
 
+  ruby_version_is "2.7" do
+    it 'raises SyntaxError if identifier is ending not on same line' do
+      -> {
+        eval %{<<"HERE\n"\nHERE}
+      }.should raise_error(SyntaxError)
+    end
+  end
+
+  ruby_version_is ""..."2.7" do
+    it 'prints a warning if identifier is ending not on same line' do
+      -> {
+        eval %{<<"HERE\n"\nHERE}
+      }.should complain(/here document identifier ends with a newline/)
+    end
+  end
```

Comment:
Just wording issue. It's common to write just `it warns` or `emits a warning`

Reply:
Do you mean add the message in heredoc string itself?
If so, check please. I've just added it.

Reply:
No-no. I meant the wording in the spec description - `it 'prints a warning`. It's not a problem and it's just a matter of taste .